django_project/core/settings/contrib.py

```diff
@@ -87,3 +87,6 @@
 CELERY_RESULT_SERIALIZER = 'json'
 
 BROKER_URL = 'amqp://guest:guest@%s:5672//' % os.environ['RABBITMQ_HOST']
+
+# Contact us settings
+CONTACT_US_EMAIL = 'alison@kartoza.com'
```

Comment:
Please don't put your email here

Reply:
Alright, ill keep that in mind, 
Thanks